lib/GADS/API.pm

```diff
@@ -546,6 +546,27 @@ any ['get', 'post'] => '/api/:sheet/records' => require_login sub {
     _get_records();
 };
 
+get '/api/:sheet/fields' => require_login sub {
+    my $user             = logged_in_user;
+    my $sheetname        = param 'sheet'
+        or panic __"No sheet name found";
```

Comment:
This should probably be `error`, as it is more likely to be a user error than a coding error